src/Stratis.Bitcoin.Features.Miner/Controllers/StakingRPCController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Security;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Controllers;
+using Stratis.Bitcoin.Features.Miner.Interfaces;
+using Stratis.Bitcoin.Features.Miner.Models;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Miner.Controllers
+{
+    /// <summary>
+    /// RPC controller for calls related to PoW mining and PoS minting.
+    /// </summary>
+    [Controller]
+    public class StakingRpcController : FeatureController
```

Comment:
Shouldn't `route` be specified here?

`[Route("api/[controller]")]`

Reply:
Oh no it shouldn't, this is the RPC controller...

Reply:
The code uses Route for controllers that inherit from `AspNetCore.MVC.Controller` (the API type controllers).

For RPC type controllers these inherit from `FeatureController`.